main.go

```diff
@@ -190,16 +196,32 @@ func initElector(cfg *config, db *sql.DB) *util.Elector {
 		os.Exit(1)
 	}
 	if cfg.restElection {
-		return util.NewElector(util.NewRestElection(), false)
+		return util.NewElector(util.NewRestElection())
 	}
 	if cfg.haGroupLockId != 0 {
+		if cfg.prometheusTimeout == -1 {
```

Comment:
I thought about this for a while. Seems like quite hard to provide "optimal" default value here, because it's tied to Prometheus scrape interval. On the other side, this particular config value has a major impact on persisting the data so I think it's the best to explicitly ask from users to set it.